src/vue/modules/requests/asset-creation-requests/store/index.js

```diff
@@ -0,0 +1,62 @@
+import { AssetCreationRequest } from '../wrappers/asset-creation-request'
+
+import { base } from '@tokend/js-sdk'
+
+import { types } from './types'
+import { api } from '../_api'
+
+const HORIZON_VERSION_PREFIX = 'v3'
+
+export const state = {
+  accountId: '',
+  assetCreationRequests: [],
+}
+
+export const mutations = {
+  [types.SET_ACCOUNT_ID] (state, accountId) {
+    state.accountId = accountId
+  },
+  [types.SET_ASSET_CREATION_REQUESTS] (state, assetCreationRequests) {
```

Comment:
same about over declarative name. we have the scoped namespace of the module. it's very obvious that by `request` we mean exactly `asset_creation` requests

Reply:
Renamed